app/lib/synthetic_study_populator.rb

```diff
@@ -148,7 +147,7 @@ def self.add_files(study, study_config, study_folder, user)
     end
   end
 
-  def self.process_expression_file_params(study, file_info)
+  def self.process_expression_file_params(_study, file_info)
```

Comment:
Non-blocking: since the `study` parameter is actually unused, rather than prefacing with the `_` I'd simply remove it here and on line 124.  These are the only two places it's referenced.